velox/type/Type.h

```diff
@@ -698,6 +698,9 @@ class RowType : public TypeBase<TypeKind::ROW> {
 
   std::string toString() const override;
 
+  // Print children name and type as comma separated pairs.
```

Comment:
typo: "children name and type" -> "child names and types"

Use /// for method comments.

Reply:
Shouldn't it be `children names and types`?